csrc/multidevice/executor.h

```diff
@@ -12,13 +12,19 @@
 #include <fusion.h>
 #include <fusion_segmenter.h>
 #include <host_ir/executor.h>
+#include <host_ir/lower.h>
 #include <ir/cloner.h>
 #include <multidevice/communication.h>
 #include <multidevice/communicator.h>
 #include <multidevice/multidevice.h>
 
 namespace nvfuser {
 
+struct MultiDeviceExecutorParams {
+  hir::HostIrEvaluatorParams executor = hir::HostIrEvaluatorParams();
+  HostIrLowerParams lower = HostIrLowerParams();
```

Comment:
```suggestion
  hir::HostIrEvaluatorParams executor;
  HostIrLowerParams lower;
```
unless this is to work around a clangtidy?